### ISSUE TITLE
Sample with Field Analyses print blank sticker

### ISSUE THREAD

**Issue by matheusjulio96** [senaite/senaite.core#2383]
## Description
Maybe i am missing something, i am still testing the software. When i try to print a sample that has Field Analyses the sticker appears with the row Field Analyses and the barcode is hidden out of bound. Sticker template is Code 128 1x72mm

## Reproduction guide
Create a sample that has field analyses, click on the large sticker button, choose sticker template code 128 1x72mm

## Observed behaviour
The preview shows an sticker without barcode, and the print generates a blank pdf

## Expected behaviour
The preview show a sticker well formated with barcode, and the print generate a pdf with content

## Versions

- OS: Linux

- senaite.core: `master`
- senaite.impress: `master`
- senaite.core.listing: `master`
- senaite.core.supermodel: `master`

## Screenshot
![image](https://github.com/senaite/senaite.core/assets/29152563/d3faa7b8-f3e9-46cd-8889-81bde3d68647)
![image](https://github.com/senaite/senaite.core/assets/29152563/76a5eb9e-2bf1-4f68-ba87-c1787ddd0e41)

**Comment by ramonski** [comment]
Hi @matheusjulio96,
I can not reproduce the bug with the latest version that runs on Docker.

<img width="1106" alt="sticker 2023-09-12 10 AM-19-06" src="https://github.com/senaite/senaite.core/assets/713193/7945bd51-18e1-403d-b167-34c28c378fb9">

Do you have a custom SENAITE installation?
A possible reason might be a missing font package on your server/workstation.

Please try again with the latest Docker image to see if the issue  is still there.

**Comment by matheusjulio96** [comment]
Right. I see that the sticker in your screenshot doesnt have the row 'Field Analyses' like mine. I'll try again

**Comment by ramonski** [comment]
Ah yes, I see now. The whole barcode disappeared because of the additional row.
I try again as well to see if I can reproduce it like this

**Comment by ramonski** [comment]
I can confirm that it does now work if the additional row "Field Analyses" is shown and the barcode overflows in the preview.
